core/core_modules/kpm.js

```diff
@@ -261,6 +280,46 @@ var git = require.once('../git.js'),
                 }
             }.bind(this));
         }.bind(this));
+    },
+    refreshModuleTable = function (url, callback) {
+        var hr = 3600000;
+        if (moduleTable.lastUpdated != null && new Date() - moduleTable.lastUpdated < hr) {
+            return callback(url);
+        }
+
+        request.get('https://raw.githubusercontent.com/wiki/mrkno/Kassy/KPM-Table.md', function (error, response) {
+            if (response.statusCode === 200 && response.body) {
+                var b = response.body;
+                if (b && b.length > 0) {
```

Comment:
Worst case: it refuses to work with the table and we do a git revert on the wiki and it magically starts working again